content/rs/getting-started/getting-started-active-active.md

```diff
@@ -92,10 +92,10 @@ Now we have two Redis Enterprise Software clusters with FQDNs
     1. In the **participating clusters** list, add the address and admin credentials for:
         - `https://cluster1.local:9443` - the cluster you are currently connected to
         - `https://cluster2.local:9443` - the other cluster
-    <!-- Also in create-crdb.md -->
+
     1. In the **Database clustering** option, either:
```

Comment:
"In" strikes me as the wrong preposition here. We usually don't "In the X option". Maybe "For"?

Reply:
This is a comment to indicate that the same content is used in another article. It doesn't show in the HTML.